framework/src/testing/fixtures/genesis_block.ts

```diff
@@ -35,14 +35,17 @@ export const createGenesisBlockWithAccounts = <T = AccountDefaultProps>(
 		delegate =>
 			(({
 				...delegate,
-				address: Buffer.from(delegate.address, 'hex'),
+				address: delegate.address,
 			} as unknown) as PartialAccount<T>),
 	);
+	// Set genesis block timestamp to 2 months in past relative to current date
+	const timestamp = Math.floor(new Date().setMonth(new Date().getMonth() - 2) / 1000);
```

Comment:
wouldnt 2 month too much? if i want to make a block on top of the block, it will have 2 month worth of the missed blocks?